frontend/src/components/Home.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google LLC
```

Comment:
I'm a bit confused about licensing here. This one mentions Google Copyright, Apache as well as MIT license. Do all of these apply?

Reply:
Google Copyright and Apache are coming from here (https://github.com/googleinterns/step-2020-recipe-finder/blob/master/LICENSE) 

MIT Licenses are for React (https://github.com/facebook/react/blob/master/LICENSE) and React Bootstrap (https://github.com/react-bootstrap/react-bootstrap/blob/master/LICENSE)